### PR TITLE
fix poi icons, dialog

### PR DESCRIPTION
**Pull request by alisa911**

### CODE REVIEW

**Comment by alisa911** [comment]
https://github.com/osmandapp/OsmAnd-Issues/issues/1903

File: map/src/context/PoiManager.js [hunk 1]
```diff
@@ -61,18 +84,28 @@ function getIconNameForPoiType(iconKeyName, typeOsmTag, typeOsmValue, iconName)
     }
 }
 
+function formattingPoiFilter(type, rename) {
+    if (type) {
+        if (rename) {
+            type = poiFilters[`${type}`].length > 1 ? poiFilters[`${type}`][1] : poiFilters[`${type}`][0];
```
Comment (RZR-UA) [inline map/src/context/PoiManager.js:90]: use **type** instead of ${type}

File: map/src/context/PoiManager.js [hunk 2]
```diff
@@ -61,18 +84,28 @@ function getIconNameForPoiType(iconKeyName, typeOsmTag, typeOsmValue, iconName)
     }
 }
 
+function formattingPoiFilter(type, rename) {
+    if (type) {
```
Comment (RZR-UA) [inline map/src/context/PoiManager.js:88]: Why not if (type && rename)?

File: map/src/drawer/components/poi/PoiTypesDialog.jsx [hunk 3]
```diff
@@ -100,6 +100,23 @@ export default function PoiTypesDialog({ dialogOpen, setDialogOpen }) {
         return categories.filter((category) => category !== 'user_defined_other' && category !== 'osmwiki');
     }
 
+    function removeUnusedFilters(filters) {
+        filters = filters.filter((f) => f !== 'routes');
```
Comment (RZR-UA) [inline map/src/drawer/components/poi/PoiTypesDialog.jsx:104]: Do you really need to modify function parameter and then return it also? If not, just use **return filters.filter(...)**

File: map/src/drawer/components/poi/PoiTypesDialog.jsx [hunk 4]
```diff
@@ -186,12 +203,12 @@ export default function PoiTypesDialog({ dialogOpen, setDialogOpen }) {
...
                                 />
                             }
-                            label={`${category}`}
+                            label={getLabel(category)}
                         />
                     ))}
                 </div>
                 <Grid container spacing={2}>
-                    {ctx.poiCategory?.filters.map((item, key) => (
+                    {removeUnusedFilters(ctx.poiCategory?.filters).map((item, key) => (
```
Comment (RZR-UA) [inline map/src/drawer/components/poi/PoiTypesDialog.jsx:211]: Need key for Grid (as 1st element under map-cycle)

**Review by RZR-UA — CHANGES_REQUESTED** [review]

**Review by RZR-UA — APPROVED** [review]

**Pull request merged**